tests/Mvc/TestController.php

```
@@ -37,7 +37,7 @@ public function partial( array $Parameters, ?Request $Request ) : string
 	public function no_request( array $Parameters, ?Request $Request )
```

PHPMD (2.15.0)

Avoid unused parameters such as '$Parameters'. (undefined)

(UnusedFormalParameter)